docs/docs/guides/atom-effects.md

```diff
@@ -239,6 +241,80 @@ const currentUserIDState = atom({
 });
 ```
 
+### Asynchronous
+
+If your persisted data needs to be `await`ed, you can either create an `asynchronous` function and call that or use a `promise` in the setSelf function 
+
+- Using the promise approach you'll be able to wrap the components inside of the `<RecoilRoot/>` with a `<Suspense/>` component to show a fallback state while waiting for `Recoil` to load the persisted values.
+- This is not possible with the async await approach.
+
+#### Promise
+
```

Comment:
Explicitly mention that this will use suspense to show a fallback while the state is pending the initial value.  If the atom is later "reset" then it will revert to the default value, not the initialized value.